src/middlewares/validateDate.js

```diff
@@ -65,15 +64,16 @@ export const checkIfRequestExists = async (req, res, next) => {
       pendingRequest.push(myRequest);
     }
   });
-
-  if (pendingRequest.length !== 0) {
+  const pendingRequestLength = pendingRequest.length;
+  if (pendingRequestLength !== 0) {
     return res.status(409).json({ satatus: 409, error: 'You still have pending request just wait for approval' });
   }
 };
 
 export const validateRequestDate = (req, res, next) => {
   const insertedDate = new Date(req.body.returnDate);
-  if (insertedDate.getFullYear() < new Date().getFullYear() || insertedDate.getFullYear() > new Date().getFullYear() + 2) {
+  if (insertedDate.getFullYear() < new Date().getFullYear() || insertedDate.getFullYear()
```

Comment:
can you remove also this duplicate!

Reply:
@Niyonsengaeric That is not duplication as it seems

Reply:
Please check that code climate issue.